routes/furnitureOptions.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const { db } = require('../server/db');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
```

Comment:
Nit-pick: can we make this name a bit more verbose like `furnitureOptionsRouter`?